doc/USAGE.md

```diff
@@ -0,0 +1,572 @@
+# Tutorial: using revaultd for the first time!
+
+## Prerequisites
+
+### Understanding the Revault architecture
+Please refer to [practical-revault](https://github.com/revault/practical-revault/) for in-depth explanation of the architecure.
+
+Every stakeholder will need:
+- `revaultd`
+- `cosignerd`
+- `revault-gui` (optional)
+
+Every manager will need:
+- `revaultd`
+- `revault-gui` (optional)
+
+Also, there must be **one** `coordinatord`
+
+### Bitcoin Core version >= 0.21.0
+See https://bitcoin.org/en/download
+
+`revaultd` **won't work** with previous versions! Check your version with `bitcoin-cli --version`
+
+### Rust
+`revaultd`, `cosignerd`, `coordinatord` and `revault-gui` are written in Rust. Since we don't publish compiled binaries (yet!), you'll need to have the Rust toolchain installed to be able to compile the projects.
+
+Please refer to https://www.rust-lang.org/tools/install for instructions on how to install Rust.
+
+Double check that your rust toolchain is **at least** 1.46.0:
+```
+cargo --version
+```
+
+### Some text editor
+There are various keys that you'll need to keep during the whole configuration. Obviously a text editor is not a good choice for storing real keys, but hey, we're in regtest. Open your favorite note-taking app (even Microsoft Word is fine, or even worse, Emacs)
+
```

Comment:
/!\ this is the first time we mention this is a test environment. IMHO it should be at the very top of the document OR if it's not about the demo, then the Regtest mention should be removed.
TLDR: to me it's unclear if USAGE.md is about usage of Revaultd, or the Demo :)

Reply:
Agree, do you think it makes sense to move it to `doc/demo.md`?

Reply:
YES :tm: `doc/DEMO.md`